androidapp/app/build.gradle

```diff
@@ -57,5 +62,5 @@ dependencies {
     testImplementation 'androidx.room:room-testing:2.2.5' // Test helpers
 
     // JeroMQ (Java implementation of ZeroMQ) for connecting to the weather server
-    implementation 'org.zeromq:jeromq:0.5.2'
+    //implementation 'org.zeromq:jeromq:0.5.2'
```

Comment:
I don't think we need to comment out JeroMQ now. Are you still having issues though when it isn't commented out?

Reply:
I'll try it and check back in